src/test/test-time-util.c

```diff
@@ -663,6 +663,10 @@ static bool timezone_equal(usec_t today, usec_t target) {
 static void test_parse_timestamp_impl(const char *tz) {
         usec_t today, now_usec;
 
+        /* Invalid */
```

Comment:
maybe extend the comment: "Ensure that systemctl reboot --when=show and --when=cancel will not result in ambiguities"